neuro_data/pipeline_management/NeuroDataPipelineManagement.py

```diff
@@ -0,0 +1,181 @@
+        area : str
+
+        Returns:
+        None
+        """
+        neuron_unit_keys = (pipeline_fuse.ScanSet().Unit() & target_scan).fetch('KEY')
+        for neuron_unit_key in neuron_unit_keys:
+            neuron_unit_key['brain_area'] = area
+            pipeline_anatomy.AreaMembership().insert1(neuron_unit_key, allow_direct_insert=True)
+
+    def manually_insert_layer_for_scan(self, target_scan, layer):
+        """
+        Give a target_scan, it will label all neurons in there with the given layer
+
+        Parameters:
+        target_scan : dict(animal_id, session, scan_idx)
+        layer : str
+
+        Returns:
+        None
+        """
+        neuron_unit_keys = (pipeline_fuse.ScanSet().Unit() & target_scan).fetch('KEY')
+        for neuron_unit_key in neuron_unit_keys:
+            neuron_unit_key['layer'] = layer
+            pipeline_anatomy.LayerMembership().insert1(neuron_unit_key, allow_direct_insert=True)
+
+    # populate functions
+    def process_static_scan(self, target_scan):
+        """
+        Function that goes and check for every table that needs to be populate as well as provide an option
+        to manaully populate AreaMembership and LayerMembership, assuming that all the neurons can be label the same Area and Layer
+        if not, they the user should manually do it.
+
+        Please refer to neuro_data/notebooks/pipeline_management notebook for an example
+
+        Parameters:
+        target_scan : dict(animal_id, session, scan_idx)
+
+        Returns:
+        None
+        """
+        print('Running preprocessing checks for ', target_scan)
+
+        # Check if the scan has been processed completely
+        if (pipeline_fuse.ScanDone() & target_scan).fetch().size == 0:
+            print('[Preprocessing Check]: ' + str(target_scan) + ' Scan has not been processed yet, please look into pipeline for details')
+            return
+        else:
+            print('[Preprocessing Check]: ScanDone Check Passed')
+
+        # Check if neurons area are labeled
+        if len(pipeline_anatomy.AreaMembership() & target_scan) == 0:
+            print('[Preprocessing Check]: ' + str(target_scan) + " AreaMembership is not populated")
+            user_input = None
+            while user_input not in ['y', 'n']:
+                user_input = input('Should we manually insert if the area are known and all the same? [y/n/exit]')
+                
+                if user_input == 'y':
+                    area = input('Input area to label neurons with:')
+                    while area not in ['V1', 'LM', 'AL', 'RL', 'all-unknown']:
+                        if area == 'exit':
+                            return
+                        print('Invalid Area!')
+                        area = input('Input area to label neurons with:')
+                        
+                    self.manually_insert_area_for_scan(target_scan, area)
+                elif user_input == 'n':
+                    return
```

Comment:
i think the check for exit is suposed to be here `if user_input == 'exit': return` rather than in area but actually I don't think you need the exit option, users already have the no option which will exit

Reply:
I change the message, yeah exit should have been inside the if area and layer